docs/marks.rst

```diff
@@ -113,42 +113,101 @@ You can then run tests that are relevant only to the selected ticket:
     cd src/tests/system
     pytest --mh-config=mhc.yaml --mh-lazy-ssh -v --ticket=gh#3433
 
-@pytest.mark.tier
-=================
+@pytest.mark.importance
+=======================
 
-The `tier mark <https://github.com/next-actions/pytest-tier>`__ can
-associate a test with a specific tier.
+The `importance mark <https://github.com/next-actions/pytest-importance>`__ can
+associate a test with a level of importance. This is used by quality engineers to
+prioritize and group the test determined by the level of impact to the customer.
```

Comment:
"and group the test**s**"?

Reply:
"A group of critical tests", but it's implied, removing it.